catroid/src/main/java/org/catrobat/catroid/pocketmusic/ui/NoteView.java

```diff
@@ -48,10 +52,22 @@ public void onClick(View v) {
 	}
 
 	private int getToggledColor() {
-		return toggled ? blackColor : whiteColor;
+		if (toggled) {
+			return toggledColor;
+		} else {
+			return getUntoggledColor();
+		}
 	}
 
 	public boolean isToggled() {
 		return toggled;
 	}
+
+	public int getUntoggledColor() {
+		return untoggledColor;
+	}
+
+	public void setUntoggledColor(int untoggledColor) {
```

Comment:
No, this method gets removed with MUS-196, as the color is then replaced with an image.
I'll fix this now. :+1: